fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -582,31 +592,55 @@ private void handleSetStmt() {
     }
 
     // Process a select statement.
-    private void handleQueryStmt() throws Exception {
+    private boolean handleQueryStmt() throws Exception {
```

Comment:
Add comment to explain the return value.

Reply:
3ks,done